docs/public/installing-odo.adoc

```diff
@@ -46,28 +46,28 @@ The following section describes how to install `odo` on different platforms via
 
 === Binary installation
 
-. Download the latest link:https://mirror.openshift.com/pub/openshift-v4/clients/odo/latest/odo-windows-amd64.exe[`odo.exe`] file.
-. Add the location of your `odo.exe` to your `GOPATH/bin` directory.
+. Download the latest link:https://mirror.openshift.com/pub/openshift-v4/clients/odo/latest/odo-windows-amd64.exe[`odo-windows-amd64.exe`] file.
+. Rename the downloaded file to `odo.exe`
+. Add the location of your `odo.exe` to your `GOPATH/bin` directory. For details on how to set `GOPATH` see link:https://golang.org/doc/gopath_code.html#GOPATH[`The GOPATH environment variable`]
 
 === Setting the `PATH` variable for Windows 10
 
 Edit `Environment Variables` using search:
+The following examples demonstrates how to set up a path variable. Your binaries can be located in any location, but this example uses C:\Go\bin as the location.
 
 . Click *Search* and type `env` or `environment`.
 . Select *Edit environment variables for your account*.
 . Select *Path* from the *Variable* section and click *Edit*.
-. Click *New* and type `C:\go-bin` into the field or click *Browse* and select the directory, and click *OK*.
+. Click *New* and type `C:\Go\bin` into the field or click *Browse* and select the directory, and click *OK*.
```

Comment:
also please dont expect that the user would have `Go` installed so suggest on creating a `C:\Go\bin` if there isn't one already.